app/src/main/java/com/example/concordia_campus_guide/Models/RoomModel.java

```diff
@@ -83,6 +86,6 @@ public void setFloorCode(@NonNull String floorCode) {
     }
 
     public String getDisplayName(){
-        return roomCode;
+        return floorCode + " " + roomCode;
```

Comment:
This is ok for now, but we should definitely discuss how a room should be displayed. According to the prototypes, it looks like it should be building code followed by room code (e.g H937).